site/content/en/blog/releases/1.42.0.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "1.42.0 - Added Kubernetes Security contexts, bug fixes and more!"
+linktitle: "1.42.0"
+date: "2024-07-16"
+---
+
+This is the 1.42.0 release of Agones.
+
+In this release we’ve added support for <a href="https://kubernetes.io/docs/tasks/configure-pod-container/security-context/" data-proofer-ignore>Kubernetes Security Contexts</a> for all Agones containers, updated our documentation, and moved work forward on <a href="https://github.com/googleforgames/agones/issues/3771" data-proofer-ignore>In-place Agones Upgrades</a>  and <a href="https://github.com/googleforgames/agones/issues/3008" data-proofer-ignore>Scheduled Autoscalers</a>.
+
+**Note**: In this release we also removed the ConnectAsync method from the C# SDK, since it wasn’t providing any functionality for several releases. There’s no need to upgrade SDK’s at this time, but be aware that if you do, this function call will need to be removed from your code.
+
+- **Enhanced Security Context in Agones Containers**:
+Security enhancements have been implemented in the Agones Helm chart to support configuring runAsUser, runAsGroup, and allowPrivilegeEscalation for core components (controller, extensions, ping, allocator), ensuring secure container operations. Additionally, default security configurations have been hardcoded into the game server sidecar template, enhancing overall container security.
+
+
+- **Documentation Enhancements**:
+We updated the client sdk page to include that  Counters and Lists have been implemented in the Node.js SDK. Additionally, clarification has been provided that the UpdateList REST documentation, specifically for how this pertains to the Local SDK Server.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.42.0" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
+- Add security context to Agones containers by @peterzhongyi in https://github.com/googleforgames/agones/pull/3856
+- Add Security Context to game server sidecar by @peterzhongyi in https://github.com/googleforgames/agones/pull/3869
+- Drop CountsAndLists Data from the Fleet and Game Server Set When the Flag is False by @igooch in https://github.com/googleforgames/agones/pull/3881
+- Adds tests to confirm that Fleet, Fleet Autoscaler, and Fleet Allocation apply defaults code is idempotent by @igooch in https://github.com/googleforgames/agones/pull/3888
+- feat: Add CRD Changes and Feature Flag for chain policy by @indexjoseph in https://github.com/googleforgames/agones/pull/3880
+
```

Comment:
Please add **Breaking changes** and **Security fixes** (I believe they are also in the template).

https://github.com/googleforgames/agones/blob/a4576c60ece35d4a75bf8b1822652e7bce65353d/docs/governance/templates/release.md?plain=1#L11